### PR TITLE
added more exercises and solutions for shell scripting

### PR DESCRIPTION
**Pull request by Brandonawan**
I've added 2 more exercises and solutions to the shell scripting section, working on more..

i would love if this pull request is merge thank you, @bregman-arie @spuder

### CODE REVIEW

**Comment by spuder** [comment]
Thanks for sharing. Unfortunately I'm not a maintainer and I can't approve.

**Comment by Brandonawan** [comment]
Please who's the maintainer? Let my pull request be merge.
Thank you!..

**Comment by IsaacDeodato** [comment]
I found your github link on twitter. Excellent content my friend. Can I
help with anything?

Att,


Em qua., 8 de mar. de 2023 19:38, Brandon Awan ***@***.***>
escreveu:

**Comment by apiviuw** [comment]
ProTip! Tambahkan komentar ke baris tertentu di bawah

File: topics/shell/directories_check.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Directories Check
+
+### Objectives
+
+1. write a script to Check if directory exists.
```
Comment (andrzejsydor) [inline topics/shell/directories_check.md:5]:
```suggestion
1. Write a script to Check if directory exists.
```

Comment (Brandonawan) [reply topics/shell/directories_check.md]: Thank you @andrzejsydor just made the correction.. 
I think it's OK, for a approval now.
Thank you..

File: topics/shell/README.md [hunk 2]
```diff
@@ -13,9 +13,11 @@
 |Count Chars|Input + While Loops|[Exercise](count_chars.md)|[Solution](solutions/count_chars.md) | Basic
 |Sum|Functions|[Exercise](sum.md)|[Solution](solutions/sum.md) | Basic
 |Number of Arguments|Case Statement|[Exercise](num_of_args.md)|[Solution](solutions/num_of_args.md) | Basic
+|Directories Check |Misc|[Exercise](directories_check.md)|[Solution](solutions/directories_check.md) | Basic
 |Empty Files|Misc|[Exercise](empty_files.md)|[Solution](solutions/empty_files.md) | Basic
 |Directories Comparison|Misc|[Exercise](directories_comparison.md)|[Solution](solutions/directories_comparison.md) | Basic
 |It's alive!|Misc|[Exercise](host_status.md)|[Solution](solutions/host_status.md) | Intermediate
+|Directories Check |Misc|[Exercise](packages_install.md)|[Solution](solutions/packages_install.md) | Advance
```
Comment (bregman-arie) [inline topics/shell/README.md:20]: I think you meant giving it a different exercise name, no?

File: topics/shell/directories_check.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+## Directories Check
+
+### Objectives
+
+1. write a script to Check if a directory exists.
```
Comment (bregman-arie) [inline topics/shell/directories_check.md:5]: s/write a script to Check if a directory exists./Write a script to check if a directory exists./

**Review by bregman-arie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**